python_ta/debug/snapshot.py

```diff
@@ -108,15 +106,14 @@ def process_value(val: Any) -> int:
                     attr_id = process_value(attr_val)
                     attr_ids[attr_name] = attr_id
                 value_entry = {
-                    "isClass": True,
-                    "name": type(val).__name__,
+                    "type": ".class",
+                    "name": val.__class__.__name__,
```

Comment:
I'm not sure why you changed the call to `type` here

Reply:
I realize now that the type call works directly, so have changed it back for consistency!